src/osd/osd_types.cc

```diff
@@ -5096,20 +5096,6 @@ void ScrubMap::decode(bufferlist::iterator& bl, int64_t pool)
   ::decode(valid_through, bl);
   ::decode(incr_since, bl);
   DECODE_FINISH(bl);
-
-  // handle hobject_t upgrade
-  if (struct_v < 3) {
-    map<hobject_t, object, hobject_t::BitwiseComparator> tmp;
-    tmp.swap(objects);
```

Comment:
@athanatos You changed it to hobject_t::ComparatorWithDefault, so it would build.  If I reset to yours, I'll have to start all the testing over again.